packages/cli/src/commands/sandbox/sandbox_command.ts

```diff
@@ -196,6 +219,31 @@ export class SandboxCommand
           boolean: true,
           global: false,
         })
+        .option('stream-function-logs', {
+          describe:
+            'Whether to stream function execution logs. Default: false. Use --function-names to filter for specific functions to stream logs from',
+          boolean: true,
+          global: false,
+          group: 'Options to configure streaming of lambda function logs',
+        })
+        .option('function-name', {
```

Comment:
We are keeping the extra argument for filtering, but it's now accepts a regex and is called `logs-filter`. This will be applicable for any cloudwatch logs we extend to in future (such as data resolver logs)